models/order.js

```diff
@@ -68,6 +70,18 @@ orderSchema.statics.updateStatus = function(orderId, status) {
   });
 }
 
+orderSchema.statics.setClosed = function(orderId, transactionId) {
+  const status = constants.CLOSED;
+  const transaction = transactionId;
+  return this.findOneAndUpdate(
+    { id: orderId },
+    { status, transaction },
+    { new: true }
+  )
+    .then((order) => populateEntries(toOrder(order)))
+    .then(populateTransaction);
```

Comment:
Handling population of transactions in next PR, when I need to use that data for the actual closed order page.